doc/release-notes-15454.md

```diff
@@ -0,0 +1,6 @@
+Wallet
+------
+
+Bitcoin Core will no longer create an unnamed `""` wallet by default when no wallet is specified on the command line or in the configuration files.
```

Comment:
> Bitcoin Core will no longer create an unnamed `""` wallet by default.

Just stop here. It won't make a default wallet of any kind, whether or not you specify one specifically.

Reply:
I think you can make an unnamed wallet with `createwallet`

Reply:
> by default

is the key phrase here. It makes nothing by default.

Reply:
Meh. Release notes can always be edited when they go up on the dev wiki.

Reply:
Ok we'll just hope our readers are deficient in English ;P

Reply:
maybe just leave out "by default"?